pkgs/by-name/su/suricata/package.nix

```diff
@@ -121,13 +120,11 @@ stdenv.mkDerivation rec {
       "--enable-unix-socket"
       "--localstatedir=/var"
       "--sysconfdir=/etc"
+      "--with-libhs-includes=${vectorscan}/include/hs"
```

Comment:
Use lib.getDev here in case vectorscan gets multiple outputs? (I've been bitten by this more than once.)

Reply:
Done (and `getLib` for the library below)